.github/workflows/emails.yml

```diff
@@ -0,0 +1,26 @@
+name: Email templates/assets sync
+on:
+  push:
+    branches:
+    - master
```

Comment:
The branching is a bit awkward; it would be weird if we had code depending on the templates in `master` but a change in `development` updated them.

Reply:
Hmm this is a bit tricky, should we have separate templates for master and development then?

Reply:
Changed to only push for master, I think this makes most sense.

Reply:
Yeah, it's definitely a weird place. We can sync to a different postmark server ("staging") for development if we want?

Reply:
oh yeah this actually makes most sense, let's create a staging server

Reply:
added emails-staging workflow for the postmark server. Also that server will be used by all test isntances on k8s.